### PR TITLE
Refactor folder structure and general style

### PR DESCRIPTION
**Pull request by rafalages**
It improve some stuff:
- Optimize Hero Image
- Use logo as a svg
- Adds a responsive behavior to the hero element
- A rework in styles structure
- Adopt BEM in the stylesheets
- Organisation of website folder
- "New site" folder was removed and merged in "website"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Megadraft [![Build Status](https://secure.travis-ci.org/globocom/megadraft.png?branch=master)](https://travis-ci.org/globocom/megadraft) [![npm version](https://img.shields.io/npm/v/megadraft.svg?style=flat)](https://www.npmjs.com/package/megadraft)
 
-Rich Text editor built on top of draft.js (Work in Progress)
+Rich Text editor built on top of [Facebook's draft.js](git@github.com:rafalages/megadraft.git) (Work in Progress)
```
Comment (vierno) [inline README.md]: Use the url of the original repo, please.

**Comment by ricobl** [comment]
Great contribution @rafalages! Thanks!

Can you please rebase? I believe we should have two commits: one for the structure refactoring and another for the README.

**Comment by rafalages** [comment]
Yes, sure!

**Comment by raphamorim** [comment]
LGTM :)

**Pull request merged**